external/kafka/src/test/resources/test_kafka.conf

```diff
@@ -0,0 +1,31 @@
+kafka {
+  consumer {
+    zookeeper.connect = "127.0.0.1:2181"
+    # consumer topics separated by ","
+    topics = "topic1"
+    client.id = "gearpump-app"
+    socket.timeout.ms = 30000
+    socket.receive.buffer.bytes = 65536
+    fetch.message.max.bytes = 1048576
+    emit.batch.size = 100
+    fetch.threshold = 5000
+    fetch.sleep.ms = 100
+  }
+
+  producer {
+    topic = "topic2"
+    metadata.broker.list = "127.0.0.1:9092"
+    producer.type = "sync"
+    request.required.acks = "1"
+    emit.batch.size = 100
+  }
+
+  storage.replicas = 1
+
+  grouper.factory.class = "org.apache.gearpump.streaming.kafka.lib.grouper.KafkaDefaultGrouperFactory"
+
+  task {
+    message.decoder.class = "org.apache.gearpump.streaming.kafka.lib.KafkaDecoder"
+    timestamp.filter.class = "org.apache.gearpump.streaming.kafka.lib.KafkaFilter"
+  }
+}
```

Comment:
but there is no much point in adding "test_kafka.conf" since "KafkaConfigSpec.scala" is intented to verify "conf/kafka.conf".